reporting/api/src/main/java/org/jboss/windup/reporting/service/ApplicationReportService.java

```diff
@@ -9,15 +9,21 @@
 
 import com.tinkerpop.blueprints.Vertex;
 import com.tinkerpop.gremlin.java.GremlinPipeline;
+import org.jboss.windup.util.Logging;
+
+import java.util.logging.Logger;
 
 /**
  * This class provides helpful utility methods for creating and finding {@link ApplicationReportModel} vertices.
  * 
  * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
+ * @author <a href="mailto:mbriskar@gmail.com">Matej Briskar</a>
```

Comment:
I am not sure if getMainApplicationReportForFile() will not cause some problems in the future, so I rather added my name here in case it will need a change.